samples/connectedhomeip/light_switch/src/AppTask.cpp

```diff
@@ -0,0 +1,191 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#include "AppTask.h"
+
+#include "LightSwitch.h"
+#include "LEDWidget.h"
+#include "ThreadUtil.h"
+
+#include <platform/CHIPDeviceLayer.h>
+#include <support/CHIPMem.h>
+#include <support/SafeInt.h>
+
+#include <dk_buttons_and_leds.h>
+#include <logging/log.h>
+#include <zephyr.h>
+
+using namespace ::chip::DeviceLayer;
+
+LOG_MODULE_DECLARE(app);
+
+namespace
+{
+static constexpr size_t kAppEventQueueSize = 10;
+
+K_MSGQ_DEFINE(sAppEventQueue, sizeof(AppEvent), kAppEventQueueSize, alignof(AppEvent));
+LightSwitch sLightSwitch;
+LEDWidget sStatusLED;
+uint8_t sLightLevel = 255; /* light brightness 0-255 */
```

Comment:
As far as I know in Zephyr/NCS there is a style of using `/* .. */` comments for the C code and `// ... ` for C++.

Reply:
right!

Reply:
as discussed, I changed all comments to `/* */`